test/boost/cql_query_test.cc

```diff
@@ -5469,7 +5469,7 @@ SEASTAR_TEST_CASE(test_null_and_unset_in_collections) {
 
         // The predicate that checks the message has to be a lambda to preserve source_location
         auto check_null_msg = [](std::source_location loc = std::source_location::current()) {
-            return exception_predicate::message_equals("null is not supported inside collections", loc);
+            return exception_predicate::message_matches(".*(null|NULL).*", loc);
```

Comment:
Looks good, except it would be good to check the error has a reasonable type and wording - and not be some ugly long "internal error" .

Reply:
The error messages are fine, I've seen most of them.